packer/timeboost/overlay/etc/systemd/system/timeboost.service

```diff
@@ -1,15 +1,15 @@
 [Unit]
 Description=Timeboost
 After=network.target nitro.service
-Requires=network-online.target
-Wants=nitro.service
+Requires=network-online.target nitro.service
```

Comment:
I wonder if the dependency to nitro has to be:

```
BindsTo=nitro.service
After=nitro.service
```

From what I understand, timeboost should not run when nitro isn't and `BindsTo` is a stronger dependency mode.